alchemy-core/src/main/java/io/rtr/alchemy/models/Treatment.java

```diff
@@ -15,6 +18,7 @@ public Treatment(String name) {
     }
 
     public Treatment(String name, String description) {
+        this.validateName(name);
```

Comment:
What about experiment names, override names, etc. as well?

Reply:
I will add validation to these, too